src/t8_schemes/t8_default/t8_default_hex/t8_default_hex_cxx.cxx

```diff
@@ -662,4 +662,69 @@ t8_default_scheme_hex_c::~t8_default_scheme_hex_c ()
    * and hence this empty function. */
 }
 
+/* each hex is packed as x,y,z coordinates and the level */
+void
+t8_default_scheme_hex_c::t8_element_MPI_Pack (const t8_element_t *elements, const int count, void *send_buffer,
+                                              const int buffer_size, int *position, sc_MPI_Comm comm) const
+{
+  int mpiret;
+  p8est_quadrant_t *quads = (p8est_quadrant_t *) elements;
+  for (int ielem = 0; ielem < count; ielem++) {
+    mpiret = sc_MPI_Pack (&(quads[ielem].x), 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].y, 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].z, 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].level, 1, sc_MPI_INT8_T, send_buffer, buffer_size, position, comm);
```

Comment:
Replace `sc_MPI_INT` and `sc_MPI_INT8_T` by t8 mpi macros having coord, level, and type in their name

Reply:
@holke, @Davknapp, is that something that you would support as well? If so, where should I put those definitions?

Reply:
I decided against it, as with the other comment regarding combining the sum into a multiplication, `level` and `type` sc_MPI_INT8_T were combined, so they can't be named separately